README.md

```diff
@@ -167,89 +301,139 @@ env:
+    # Requirements file with contents for tests dependencies
+    - CONDA_REQUIREMENTS="dev-requirements.txt"
+```
+
+#### shared_configs/python-tester-pip.yml
 `python-tester.yml` runs any pytest tests it finds after installing the
-specified requirements.
+specified requirements via PIP using Python 3.6.
 
-#### usage:
+##### usage:
 This configuration can be added to the `test` stage of a Travis build by
 importing it in your `.travis.yml`:
 ``` yaml
 import:
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/python-tester.yml
+  - pcdshub/pcds-ci-helpers:travis/shared_configs/python-tester-pip.yml
 ```
-#### arguments:
+##### arguments:
 Requirements files can be specified by assigning values to `REQUIREMENTS` and
 `DEV_REQUIREMENTS`. `REQUIREMENTS` defaults to `requirements.txt` and
 `DEV_REQUIREMENTS` defaults to `dev-requirements.txt`. If no requirements are
 necessary, the file can be blank.
+
+Additional dependencies not specified at the requirements files above can be
+passed to the install process via the `PIP_EXTRAS` variable.
+
 ``` yaml
 env:
   global:
     - REQUIREMENTS: requirements.txt
     - DEV_REQUIREMENTS: dev-requirements.txt
+    - PIP_EXTRAS="PyQt5 happi"
 ```
 
-### shared_configs/doctr-upload.yml
+#### shared_configs/docs-build.yml
 `docs-build.yml` runs through the build of the package's documentation to
 ensure it works properly.
+After building the documentation it is available via the `workspaces` named
+`docs` which is used by the `doctr-upload.yml` later on for upload.
+As of now, the docs build task uses a conda environment since not all of our
+dependencies are available through PIP. This can be revisited in the future
+once the packages are correctly available.
 
-#### usage:
+##### usage:
 This configuration can be added to the `test` stage of a Travis build by
 importing it in your `.travis.yml`:
 ``` yaml
 import:
   - pcdshub/pcds-ci-helpers:travis/shared_configs/docs-build.yml
 ```
 
-#### arguments:
-- A docs requirements file can be specified by assigning a value to
+##### arguments:
+- One or more docs requirements file can be specified by assigning a value to
   `DOCS_REQUIREMENTS`, which defaults to `docs-requirements.txt`.
 - The folder containing the documentation can be specified by assigning a value
   to `DOCS_FOLDER`, which defaults to `docs`.
 ``` yaml
 env:
   global:
-    - DOCS_REQUIREMENTS: docs-requirements.txt
+    - DOCS_REQUIREMENTS="dev-requirements.txt requirements.txt"
     - DOCS_FOLDER: docs
+    - CONDA_EXTRAS="pip pyqt=5 happi"
 ```
 
-### shared_configs/anaconda-upload.yml
-`anaconda-upload.yml` builds the package according to the `conda-recipe` and
-uploads it to the `pcds-dev` channel on Anaconda Cloud. If the build was
-triggered by a tag, the package will additionally be uploaded to the `pcds-tag`
-channel.
 
-#### usage:
+### Deploy Jobs
+
+#### shared_configs/anaconda-upload.yml
+`anaconda-upload.yml` uses the `workspaces` named `conda` to leverage the
```

Comment:
```suggestion
`anaconda-upload.yml` uses the workspace named `conda` to leverage the
```
Same as above. I guess there are more instances of `` `workspaces` `` than `workspace`. Your choice